### PR TITLE
CRAB-24704: Allow alternate unit names simple cases

### PR DESCRIPTION
**Pull request by botondlaczko**

### CODE REVIEW

File: src/main/java/javax/measure/unit/NonSI.java [hunk 1]
```diff
@@ -196,6 +220,20 @@ public static NonSI getInstance() {
      */
     public static final Unit<Length> COMPUTER_POINT = PIXEL;
 
+
+    /**
+     * New unit of length equal to <code>>meter</code>
+     */
+    private static final Unit<Length> GIGA_MICRON = new AlternateUnit<>("gmicron",
+            METER);
```
Comment (dustin-johnson) [inline src/main/java/javax/measure/unit/NonSI.java:228]: Why is this needed?

**Comment by dustin-johnson** [comment]
Why can't these additions be made in appserver's ScalarUnits.java?

**Comment by rsmith20** [comment]
> Why can't these additions be made in appserver's ScalarUnits.java?

Boti was deriving from my solution for mbar, hPa and I did these in jscience, although ScalarUnits makes more sense. They were initially in Jscience because I was doing a more complex solution that required Jscience tinkering and I didn't think to port after reducing the complexity. I these grouping of changes should stay together though, so I think the dekamicrobar and these changes should get put into ScalarUnits.

File: src/main/java/javax/measure/unit/NonSI.java (hunk 1 above)
Comment (rsmith20) [reply src/main/java/javax/measure/unit/NonSI.java:228]: You cannot directly give alternate names to non-base units (such as micrometer/micron). But you can make a non-sense alternate of the base unit, and derive the aliased name from that non-sense unit.

File: src/main/java/javax/measure/unit/NonSI.java [hunk 2]
```diff
@@ -95,6 +95,30 @@ public static NonSI getInstance() {
...
+     * New unit of length equal to <code>>ppm</code>
+     */
+    private static final Unit<Miscellaneous> PARTS_PER_MILLION_VOLUME = new AlternateUnit<>("ppmv",
+            Miscellaneous.ppm);
+
+    /**
+     * A unit of length equal to <code>>pmmv</code>
+     * (standard name <code>ppm</code>).
+     */
+    public static final Unit<Miscellaneous> PPMV = nonSI(PARTS_PER_MILLION_VOLUME);
```
Comment (rsmith20) [inline src/main/java/javax/measure/unit/NonSI.java:108]: I don't think it is necessary to define these separately.

File: src/main/java/javax/measure/unit/NonSI.java (hunk 1 above)
Comment (rsmith20) [reply src/main/java/javax/measure/unit/NonSI.java:228]: Side note: we would probably want a different symbol than gmicron. These units can unfortunately be instantiated in Seeq, so we want to give them names that won't occur by accident.

**Comment by rsmith20** [comment]
Concept makes sense. Based on my other comment I do see this change as more properly belonging in ScalarUnits within appserver.

File: src/main/java/javax/measure/unit/NonSI.java (hunk 1 above)
Comment (botondlaczko) [reply src/main/java/javax/measure/unit/NonSI.java]: made AlternateUnit public and moved this implementations into ScalarUnits class
